spruce/Dockerfile

```diff
@@ -0,0 +1,6 @@
+FROM golang:1.5.1-alpine
+
+RUN apk update
+RUN apk add git
+RUN go get github.com/geofffranks/spruce
+RUN apk del git
```

Comment:
You may want to switch the last 3 commands to just 
``` RUN apk add git && go get github.com/geofffranks/spruce && apk del git```` this will produce a single layer that will produce a smaller image